src/test/java/ui/main_menu/MainMenuTest.java

```diff
@@ -0,0 +1,120 @@
+package ui.main_menu;
+
+import game.ui.SceneLauncher;
+import game.ui.game.GameScene;
+import game.ui.menu.MenuScene;
+import org.assertj.swing.fixture.FrameFixture;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import ui.util.TestComponentNames;
+
+import static org.junit.Assert.assertEquals;
+
+public class MainMenuTest {
+    private FrameFixture window;
+    private MenuScene menuScene;
+    private SceneLauncher launcher;
+
+    @Before
+    public void setUp() {
+        launcher = new SceneLauncher();
+        menuScene = (MenuScene)launcher.getCurrentScene();
+        window = new FrameFixture(menuScene);
+        window.show();
+    }
+
+    @Test
+    public void startGameWithZeroPlayers(){
+        //ARRANGE
+
+        //ACT
+        window.button(TestComponentNames.START_GAME).click();
+
+        //ASSERT
+        assertEquals( launcher.getCurrentScene(), menuScene);
+    }
+
+    @Test
+    public void startGameWithAddingPlayerWithTooLongName(){
+        //ARRANGE
+        var TOO_LONG_NAME = "12345678901";
+
+        //ACT
+        window.button(TestComponentNames.NEW_PLAYER).click();
+        var optionPaneFixture = window.optionPane();
```

Comment:
These 3 lines come up often, we could move them to a private function with a string param for `enterText`

Reply:
Ok, refactored the code.